client/src/components/ElementEditor/FormBuilder.js

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
+import classnames from 'classnames';
+import FormBuilderLoader from 'containers/FormBuilderLoader/FormBuilderLoader';
+import Config from 'lib/Config';
 
-function FormBuilder() {
-  return <div className="element-editor-formbuilder">Imagine some fancy form builder here!</div>;
+function FormBuilder(props) {
+  const classNames = classnames(
+    'element-editor-formbuilder',
+    props.extraClasses
+  );
+  const baseURL = Config.get('baseUrl').replace(/\/$/, '');
+  return (<div className={classNames} onClick={props.onClick} role="presentation">
+    <FormBuilderLoader
+      formTag="div"
+      schemaUrl={`${baseURL}/admin/elemental-area/schema/elementForm/${props.elementId}`}
+      identifier="element"
+    />
+  </div>);
```

Comment:
Do we actually need this component at all now? I had imagined it would get replaced entirely by FormBuilderLoader. Assuming you can pass extra classes into it would that be enough to remove this?

Reply:
I've renamed it to InlineEditForm to remove ambiguity - I'm happy now that it exists